adserver/admin.py

```diff
@@ -502,7 +505,7 @@ class ClickAdmin(AdBaseAdmin):
 
     # Browser Family and OS Family are not in the ``ViewAdmin.list_filter``
     # because they require a ``SELECT DISTINCT`` across the whole table
-    list_filter = ("is_mobile", "is_bot", "browser_family", "os_family")
+    list_filter = ("is_mobile", "publisher", "browser_family", "os_family")
```

Comment:
I replaced the `is_bot` filter which is totally useless anyway. We already exclude bots from counting toward views and clicks anyway. That field can probably just be deleted on the model.